### PR TITLE
✨ PIP_COMPILE_DISABLE

### PR DESCRIPTION
**Pull request by juftin**
Disables any changes to a lockfile if the `PIP_COMPILE_DISABLED` env var is set

closes https://github.com/juftin/hatch-pip-compile/issues/19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -313,6 +313,18 @@ Using the default hatch configuration, dev dependencies listed in your
 from the lockfile you must remove them from the `default` environment
 on your `pyproject.toml` / `hatch.toml` file.
 
+### Disabling Changes to the Lockfile
+
+In some scenarios, like in CI/CD you want the tool to manage the installation
+of an environment, but you don't want the lockfile to be or changed without
```
Comment (oprypin) [inline README.md:319]: some typo in this line?

File: README.md [hunk 2]
```diff
@@ -313,6 +313,18 @@ Using the default hatch configuration, dev dependencies listed in your
...
+knowing. If you set the `PIP_COMPILE_DISABLED` environment variable to
```
Comment (oprypin) [inline README.md:320]: I don't have any good arguments for this right now but for some reason PIP_COMPILE_DISABLE seems like a more natural name to me than PIP_COMPILE_DISABLED

Comment (juftin) [reply README.md:320]: And one letter less too! Works for me.

**Review by oprypin — APPROVED** [review]

**Pull request merged**